src/test/java/org/broadinstitute/hellbender/tools/exome/FilterSamplesIntegrationTest.java

```diff
@@ -0,0 +1,285 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Integration tests for {@link FilterSamples}.
+ *
+ * @author Valentin Ruano-Rubio &lt;valentin@broadinstitute.org&gt;
+ */
+public class FilterSamplesIntegrationTest extends CommandLineProgramTest {
+
+    @Override
+    public String getTestedClassName() {
```

Comment:
I mean without a compilation error. I think that in general one should avoid relaying on name convention based reflection. If it was me I rather would rather have left that method as abstract in the parent class.